client/src/components/RatingsAndReviews/AddReviewModal.jsx

```diff
@@ -11,22 +27,142 @@ class AddReviewModal extends React.Component {
+            </legend>
+          <input
+            type="radio"
+            required
+            name="recommendation"
+            id="recommend-yes"
+            value="yes"
+            checked={this.state.recommendation === 'yes' || false}
+            onChange={this.onChangeHandler} />Yes
+          <label htmlFor="recommend-yes" value="Yes"/>
+          <input
+          type="radio"
+          required
+          name="recommendation"
+          id="recommend-no"
+          value="no"
+          checked={this.state.recommendation === 'no' || false}
+          onChange={this.onChangeHandler}/> No
+          <label htmlFor="recommend-no" value="No"/>
+        </fieldset>
+        <label>
+          Review Summary:
+          </label>
+        <input
+          type="text"
+          id="review-summary"
+          name="reviewSummary"
+          defaultValue={this.state.reviewSummary}
+          maxLength="60"
+          data-testid="review-summary"
+          onChange={this.onChangeHandler}
+        />
+        <br></br>
+        <label>
+          <abbr title="This field is mandatory"
+            aria-label="required"> * </abbr>
+            Review:
+        </label>
+        <textarea
+          className="form-body-text"
+          id="review-body-text"
+          name="reviewBody"
+          value={this.state.reviewBody}
+          required
+          maxLength="1000"
+          onChange={this.onChangeHandler}
+        />
+        <WordCount text={this.state.reviewBody} />
+        {/* <input type="file" name="review-photo" multiple /> */}
+        <label>
+          <abbr title="This field is mandatory"
+            aria-label="required"> * </abbr>
+              What is your nickname?
+        </label>
+        <input
+          type="text"
+          id="review-nickname"
+          name="nickname"
+          defaultValue={this.state.nickname}
+          maxLength="60"
+          data-testid="review-nickname"
+          onChange={this.onChangeHandler}
+        />
+        <p className="form-detail-text">For privacy reasons, do not use your full name or email address</p>
+        <br></br>
+        <label>
+        <abbr title="This field is mandatory"
+            aria-label="required"> * </abbr>
+            Your Email:
+        </label>
+        <input
+          type="text"
+          id="review-email"
+          name="email"
+          defaultValue={this.state.email}
+          data-testid="review-name"
+          onChange={this.onChangeHandler}
+        />
+        <p className="form-detail-text">For authentication reasons, you will not be emailed</p>
+        <input
+          type="submit"
+          value="Submit Review"
+          name="submit"
+          data-testid="review-upload-form-submit"
+        />
+        <span id="status"></span>
+        <input type="file" onChange={this.onChangeFileHandler}/>
+        <ReviewThumbnailContainer thumbnails={this.state.selectedImageFiles} imageLoaded={this.state.imageLoaded} />
```

Comment:
What is imageLoaded supposed to be? Noticed this is the only place it is provided so kind of curious if I am missing the reference to it though.